docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+services:
+  mariadb-database:
+    image: mariadb:10.4
+    container_name: mariadb-database
+    environment:
+      MYSQL_ROOT_PASSWORD: homestead
+      MYSQL_DATABASE: homestead
+      MYSQL_USER: homestead
+      MYSQL_PASSWORD: secret
+    ports:
+      - "3307:3306"
+    restart: always
+    volumes:
+      - "./create_tables.sql:/docker-entrypoint-initdb.d/1.sql"
```

Comment:
I think we don't need to include this docker-compose file. It won't be necessary for the production environment. Did you use this for local development right?